src/components/Header.jsx

```diff
@@ -1,9 +1,9 @@
-const Header = () => {
-  return (
-    <header>
-      <h1>todos</h1>
-      <p>Items will persist in the browser local storage</p>
-    </header>
-  );
-};
+import styles from '@/styles/Header.module.css';
+
+const Header = () => (
+  <header style={styles} className={styles.header}>
+    <h1>todos</h1>
+    <p>Items will persist in the browser local storage</p>
+  </header>
```

Comment:
Your readme says the local storage was a future feature. In that case it's better to remove this paragraph till then as it leads one to a wrong conclusio.